containers/opcua-device-gateway/Dockerfile

```diff
@@ -40,14 +40,13 @@ RUN case "${VERSION}" in \
     esac \
     && wget -O opcua-device-gateway.jar "$DOWNLOAD_URL"
 
-COPY application-tenant.yaml .
+
 COPY logging.xml .
 COPY entrypoint.sh .
 
 # Default settings
 ENV OPCUA_GATEWAY_IDENTIFIER=OPCUAGateway
 ENV OPCUA_GATEWAY_NAME=OPCUAGateway
```

Comment:
This variable has been renamed, so it should be `GATEWAY_NAME` now.

Reply:
Done